Protocol.md

```diff
@@ -81,6 +82,11 @@ the funding transaction and CETs.
    * [`u64`:`feerate_per_vb`]
    * [`u32`:`cet_locktime`]
    * [`u32`:`refund_locktime`]
+   * [`offer_tlvs`: `tlvs`]
+
+[//]: # (TODO: `protocol_version` should be advertised by DLC nodes in the future)
```

Comment:
is there any precedent for this in other deployed cryptocurrency protocols? Could you reference some resources you used for this if so? 

It seems like this isn't just a problem with our protocol, but every cryptocurrency protocol that has versioning.

Reply:
Oh sorry, this is a TODO nevermind.

Reply:
I think having a protocol version is really common in any kind of protocol. It's for example used in the bitcoin p2p protocol: https://en.bitcoin.it/wiki/Protocol_documentation#version

Reply:
it would be nice to have a list of known protocol_versions. My understanding is the first one is introduced in this PRs test vectors. It would be nice to have it listed somewhere in this doc